piet-scene/src/scene/builder.rs

```diff
@@ -14,17 +14,17 @@
 //
 // Also licensed under MIT license, at your choice.
 
-use super::style::{Fill, Stroke};
-use super::{Affine, BlendMode, PathElement, Scene, SceneData, SceneFragment};
-use crate::{brush::*, ResourcePatch};
+use super::{conv, Scene, SceneData, SceneFragment};
+use crate::ResourcePatch;
 use bytemuck::{Pod, Zeroable};
-use core::borrow::Borrow;
+use peniko::kurbo::{Affine, PathEl, Shape};
+use peniko::{BlendMode, BrushRef, ColorStop, Fill, Stroke};
 use smallvec::SmallVec;
 
 /// Builder for constructing a scene or scene fragment.
 pub struct SceneBuilder<'a> {
     scene: &'a mut SceneData,
-    layers: SmallVec<[BlendMode; 8]>,
+    layers: SmallVec<[(BlendMode, bool); 8]>,
```

Comment:
I personally would find it useful to have a comment what the bool means.

Reply:
Will do.